dashboard/apps/prototype/static/prototype/js/main.js

```diff
@@ -191,24 +191,24 @@ var testRequest = {
   end_date: '2016-05-23'
 };
 
+function mapFloats(list, key) {
```

Comment:
the problem this function is trying to address is captured in this function here:
https://github.com/ministryofjustice/moj-product-dashboard/blob/feature/unit-tests/dashboard/apps/prototype/static/prototype/js/main.js#L37